JavaScript/form.js

```diff
@@ -0,0 +1,39 @@
+const form = document.querySelector('form');
+
+form.addEventListener('submit', function(event) {
+
+    const responses = [
+        document.getElementById('applicantName').value,
+        document.getElementById('applicantEmail').value,
+        document.getElementById('applicantAddress').value,
+        document.getElementById('applicantCity').value,
+        document.getElementById('applicantState').value,
+        document.getElementById('applicantZip').value,
+        document.getElementById('applicantFirst').value,
+        document.getElementById('applicantPickup').value
+    ];
+
+    let pass = true;
+
+    for (let i = 0; i < responses.length; i++) {
+        res = responses[i];
+        if (res == '') {
+            pass = false;
+        }
+        if (i === 1 && !responses[i].includes('@')) {
```

Comment:
Small change, but I would instead do 
if !responses[1].includes('@')) { 
so there is only one check in the if and maybe move it outside of the loop so it doesn't  get checked every time. 
(very minor note)